app/views/content/life-as-a-teacher/teaching-as-a-career/from-qualified-teacher-to-head-of-biology/_article.html.erb

```diff
@@ -23,3 +23,16 @@
       <p>Undeniably, teaching is hard work – but the knowledge that you have helped to motivate young people and helped shape their future brings more satisfaction than any other job I can think of.</p>
   </section>
 </div>
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render CallsToAction::MailinglistBlockComponent.new(
```

Comment:
It feels like we are making the same flexible changes across multiple changes - shouldn't we just have a block file? Or is this a limitation on html pages?